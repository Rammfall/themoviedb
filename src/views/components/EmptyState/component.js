import React from 'react'
import PropTypes from 'prop-types'
import { Empty } from 'antd'

const EmptyState = ({ title }) => (
  <div className='top-margin'>
    <Empty
      description={title}
      image={Empty.PRESENTED_IMAGE_SIMPLE}
    />
  </div>
)

EmptyState.propTypes = {
  title: PropTypes.string.isRequired
}

export default EmptyState
