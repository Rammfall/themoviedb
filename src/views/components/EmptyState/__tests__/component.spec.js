import React from 'react'
import { shallow } from 'enzyme'

import EmptyState from '../component'

describe('EmptyState', () => {
  it('matches snapshot', () => {
    const component = shallow(<EmptyState title="test" />)

    expect(component).toMatchSnapshot()
  })
})
