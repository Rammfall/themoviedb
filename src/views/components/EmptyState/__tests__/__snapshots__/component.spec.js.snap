// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState matches snapshot 1`] = `
<div
  className="top-margin"
>
  <Empty
    description="test"
    image={<Simple />}
  />
</div>
`;
